torchdata/datapipes/iter/util/plain_text_reader.py

```diff
@@ -154,6 +168,7 @@ def __iter__(self) -> Iterator[Union[D, Tuple[str, D]]]:
             stream = self._helper.skip_lines(file)
             stream = self._helper.decode(stream)
             stream = self._csv_reader(stream, **self.fmtparams)
+            stream = self._helper.as_tuple(stream)
```

Comment:
`mypy` complains about it at this line. Could you please try to fix it? Otherwise, you can add a comment at the end of this line to bypass the mypy check with `# type: ignore[assignment]`

Reply:
Done